src/ig/IGExporter.ts

```diff
@@ -1479,6 +1556,53 @@ export class IGExporter {
       }
     }
   }
+
+  // Add an extension for the R5 property IG.definition.page.source[x] if provided in configuration
+  addPageSourceExtensionForR4(
+    page: ImplementationGuideDefinitionPage,
+    configPages: ImplementationGuideDefinitionPage[]
+  ): void {
+    const extensionUrl =
+      'http://hl7.org/fhir/5.0/StructureDefinition/extension-ImplementationGuide.definition.page.source';
+    const configPage = configPages?.find(
+      p =>
+        p.nameUrl.substring(0, p.nameUrl.lastIndexOf('.')) ===
+        page.nameUrl.substring(0, page.nameUrl.lastIndexOf('.'))
+    );
+    if (configPage) {
+      if (configPage.sourceUrl) {
```

Comment:
I _think_ this is the same as `page.nameUrl` in R4 -- but I'm not 100% confident enough in that to say we should overwrite an existing `nameUrl` with a `sourceUrl`.  So... I think what you're doing is probably best.

Reply:
Similar to above, I think going off the R4 to R5 implementation, we only set sourceUrl based on the direct `source[x]` property in configuration and don't infer anything from an R4 IG. So I think that's why I added it as an extension. But similar to above, maybe this was the wrong approach.

Reply:
Similarly, this is taken care of with the approach from the comment above and fixed in [67deb44](https://github.com/FHIR/sushi/pull/1189/commits/67deb44fe0b08395742a935a5189fa6c7726e4b5).